CHANGELOG.md

```diff
@@ -9,15 +9,13 @@
 - Added distributed tracing using open telemetry APIs for table stored procedure function in `DataFrame`:
   - _execute_and_get_query_id
 - Added support for function `arrays_zip`.
-- Allow `df.plot()` and `series.plot()` to be called, materializing the data into the local client
 - Improves performance for binary column expression and df._in by avoiding unnecessary cast for numeric values. This optimization can be enabled through session.eliminate_numeric_sql_value_cast_enabled = True.
 - Improved error message for `write_pandas` when target table does not exists and `auto_create_table=False`.
 - Added open telemetry tracing on UDxF functions in snowpark.
 - Added open telemetry tracing on stored procedure registration in snowpark.
 - Added a new optional parameter called `format_json` to the `Session.SessionBuilder.app_name` function that allows to set the app name in the `Session.query_tag` in JSON format. By default, this parameter is set to `False`.
 
 #### Bug Fixes
-- Fixed a bug regarding precision loss when converting to Snowpark pandas `DataFrame` or `Series` with `dtype=np.uint64`.
```

Comment:
is this removed because of duplicate?

Reply:
Yes, and it is a snowpark pandas change, shouldn't be under snowpark python api